sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/TrackNotImplementedException.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace Tests.Diagnostics
```

Comment:
Will add the tests. I don't think we should handle sub-classes of `NotImplementedException`.